data-structures/trie/trie.py

```diff
@@ -0,0 +1,40 @@
+class trienode:
+	def __init__(self,character):
+		self.character = character
+		self.nextnodes = {}
+		self.isEnd = False
+		self.dataNode = None
+	
+class trie: # use case: username/password
+	def __init__(self):
+		self.start = trienode('')
+	def insert(self,id,data): # returns true on successful insertion (data == password)
+		temp = self.start
```

Comment:
Please use an appropriate variable name for `temp`
Refer to - https://www.quora.com/Why-is-using-a-variable-name-temp-considered-bad-practice

Reply:
Updated the variable name.

Reply:
The variable name is still `temp`. Please change it to something meaningful @GajeshS